### ISSUE TITLE
panic on running yarn after installing from scratch

### ISSUE THREAD

**Issue by rwjblue** [notion-cli/notion#196]
Steps to reproduce (on a fresh machine):

```
curl -sSLf https://get.notionjs.com | bash
# open new terminal
notion install yarn
cd ./some-project
yarn
```

Results in the following stack trace:

```
❯ RUST_BACKTRACE=1 yarn
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', libcore/option.rs:345:21
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::continue_panic_fmt
   6: rust_begin_unwind
   7: core::panicking::panic_fmt
   8: core::panicking::panic
   9: <notion_core::tool::Yarn as notion_core::tool::Tool>::new
  10: notion_core::tool::Tool::launch
  11: yarn::main
  12: std::rt::lang_start::{{closure}}
  13: std::panicking::try::do_call
  14: __rust_maybe_catch_panic
  15: std::rt::lang_start_internal
  16: main
```

**Comment by rwjblue** [comment]
After some digging, I believe that the issue is that I hadn't installed any Node versions.

The panic is caused by:

https://github.com/notion-cli/notion/blob/v0.1.4/crates/notion-core/src/tool.rs#L304-L305

That `.unwrap()` was removed as part of https://github.com/notion-cli/notion/pull/159.

**Comment by rwjblue** [comment]
Confirmed (by building master locally) that the changes in #159 made this much much better. Using the same repro steps (though building from source):

1. [install from source](https://github.com/notion-cli/notion#source-installation)
2. `notion install yarn`
3. `yarn`

Gives the following output now:

```
❯ yarn
Notion error:
No Yarn version selected.

See `notion help use` for help adding Yarn to a project toolchain.

See `notion help install` for help adding Yarn to your personal toolchain.
```

---

Now, as you can see the message isn't _quite_ correct (we do have an installed `yarn`, but no installed `node`), but at least it isn't a panic...

**Comment by charlespierce** [comment]
This was fixed by the error rework done in #328
